cli/js/process.ts

```diff
@@ -23,10 +23,11 @@ async function runStatus(rid: number): Promise<ProcessStatus> {
 
   if (res.gotSignal) {
     const signal = res.exitSignal;
-    return { signal, success: false };
+    return { success: false, code: 128 + signal, signal };
```

Comment:
`code: 128 + signal` - is there some prior art for such exit code?

Reply:
Got it from here: https://stackoverflow.com/a/39269908 or https://tldp.org/LDP/abs/html/exitcodes.html.